src/style/adwaitahelper.cpp

```diff
@@ -29,6 +29,8 @@
 
 #if ADWAITA_HAVE_X11
 #include <X11/Xlib-xcb.h>
+// POP_OS
```

Comment:
Unrelated change.

Reply:
Yes irrelevant, it was just temporary otherwise compilation fails on POP.